docs/src/main/paradox/scala/http/handling-blocking-operations-in-akka-http-routes.md

```diff
@@ -133,4 +133,4 @@ applications. It is referred to as "bulkheading" or "isolating" the bad behaving
 parts of an app. In this case, bad behaviour of blocking operations.
 
 There is good documentation available in Akka docs section, 
-[Blocking needs careful management](http://doc.akka.io/docs/akka/current/general/actor-systems.html#Blocking_Needs_Careful_Management).
+[Blocking needs careful management](http://doc.akka.io/docs/akka/current/scala/dispatchers.html#blocking-needs-careful-management).
```

Comment:
It needs to point to the the Java version or the Scala one accordingly. Basically you would need to use the `@scala[]` and `@java[]` directives.
You can read further here http://developer.lightbend.com/docs/paradox/latest/features/groups.html#inline

Reply:
Thanks for pointing that out. I'll fix it.

Reply:
Thank you for the quick response :+1: